src/main/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2021 Micro Focus or one of its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Docker registry
+ARG DOCKER_HUB_PUBLIC=docker.io
+
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/buildenv-dotnet:1.0.0
+CMD ["bash"]
+
+RUN zypper addrepo https://download.opensuse.org/repositories/home:zkrd2:mono/15.4/home:zkrd2:mono.repo
```

Comment:
I'm not dying about the `15.4` being hidden away in there.  It would be better if we found a way to dynamically update it based on the OpenSUSE version.  Also I see the version of `cafapi/buildenv-dotnet:1.0.0` that this is based on is currently `15.1` so it will have to be brought up to date also.